src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/DashboardViewModel.cs

```diff
@@ -310,6 +310,11 @@ public bool DefaultPreviewChangesFlag
             }
         }
 
+        public string GetOriginalName()
```

Comment:
>GetOriginalName [](start = 22, length = 15)

nit: This looks more like a property than a function

Reply:
Agreed, fixed.